### PR TITLE
Updated readme, added how to generate i18n en properties file/kar files/use -pl option

### PR DESCRIPTION
**Pull request by Hilbrand**
- Also put command in blocks to make them stand out more in the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,19 +94,55 @@ To improve build times you can add the following options to the command:
...
 
 Adding these options improves the build time but could hide problems in your code.
 Parallel builds are also less easy to debug and the increased load may cause timing sensitive tests to fail.
 
-To check if your code is following the [code style](https://www.openhab.org/docs/developer/guidelines.html#b-code-formatting-rules-style) run: `mvn spotless:check`
-To reformat your code so it conforms to the code style you can run: `mvn spotless:apply`
+#### Translations
+
+Add-on translations are managed via [Crowdin](https://crowdin.com/project/openhab-addons).
+The english translation is taken from the openHAB-addons GitHub repo and automatically imported in Crowdin when changes are made to the english i18n properties file.
```
Comment (jlaur) [inline README.md:111]:
```suggestion
The English translation is taken from the openHAB-addons GitHub repo and automatically imported in Crowdin when changes are made to the English i18n properties file.
```

File: README.md [hunk 2]
```diff
@@ -76,19 +94,55 @@ To improve build times you can add the following options to the command:
...
+When translations are added or updated and approved in Crowdin a pull request is automatically created by Crowdin.
```
Comment (jlaur) [inline README.md:112]:
```suggestion
When translations are added or updated and approved in Crowdin, a pull request is automatically created by Crowdin.
```

File: README.md [hunk 3]
```diff
@@ -76,19 +94,55 @@ To improve build times you can add the following options to the command:
...
+Therefor translations should not be edited in the openHAB-addons repo, but only in Crowdin.
```
Comment (jlaur) [inline README.md:113]:
```suggestion
Therefore translations should not be edited in the openHAB-addons repo, but only in Crowdin.
```

File: README.md [hunk 4]
```diff
@@ -76,19 +94,55 @@ To improve build times you can add the following options to the command:
...
+Otherwise translation are overridden by the automatic process.
+
+To fill the english properties file run the following maven command on an addon-on:
```
Comment (jlaur) [inline README.md:116]:
```suggestion
To fill the English properties file run the following maven command on an add-on:
```

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks for improving this documentation! I have posted a few proof-reading comments.

File: README.md [hunk 5]
```diff
@@ -76,19 +94,55 @@ To improve build times you can add the following options to the command:
...
+
+```shell
+mvn i18n:generate-default-translations
```
Comment (lolodomo) [inline README.md:119]: The command is not working for everybody. 
The following is apparently working for everybody: mvn org.openhab.core.tools:i18n-maven-plugin:3.2.0:generate-default-translations

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (Hilbrand) [reply README.md:119]: I've added the command you mentioned here as an alternative option in the text below in case above command doesn't work.

**Comment by lolodomo** [comment]
All these information should certainly be also included in the developer documentation.

**Review by lolodomo — APPROVED** [review]
LGTM

**Comment by lolodomo** [comment]
@jlaur : I let you the final word on this PR.

**Review by jlaur — APPROVED** [review]
Thanks!

**Pull request merged**